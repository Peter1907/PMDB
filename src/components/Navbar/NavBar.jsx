import { Link } from 'react-router-dom';
import Menu from '../Menu/Menu';
import s from './Navbar.module.css';

const NavBar = () => (
  <header className={s.header}>
    <Link className={s.homeLink} to="/">
      <h2 className={s.title}>PMDB</h2>
    </Link>
    <Menu />
  </header>
);

export default NavBar;
