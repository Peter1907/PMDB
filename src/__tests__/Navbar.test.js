import renderer from 'react-test-renderer';
import NavBar from '../_mocks_/Navbar';

it('renders correctly', () => {
  const tree = renderer
    .create(<NavBar />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
