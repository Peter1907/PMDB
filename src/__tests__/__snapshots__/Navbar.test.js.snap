// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="header"
>
  <a
    className="homeLink"
    href="/"
    onClick={[Function]}
  >
    <h2
      className="title"
    >
      PMDB
    </h2>
  </a>
  <a
    className="link one"
    href="/top-movies"
    onClick={[Function]}
  >
    Top Movies
  </a>
  <a
    className="link"
    href="/top-series"
    onClick={[Function]}
  >
    Top Series
  </a>
  <a
    className="link"
    href="/pop-movies"
    onClick={[Function]}
  >
    Popular Movies
  </a>
  <a
    className="link"
    href="/pop-series"
    onClick={[Function]}
  >
    Popular Series
  </a>
</header>
`;
